tests/run_all.sh

```diff
@@ -7,10 +7,15 @@ tests_to_run=(
     fake_image_test
     base_test
     url_test
+    imap_test
```

Comment:
Can you also add a test where the associated image is missing? In case we accidentally specify a bad or missing image in the image map.